doc/source/change_proposal_process.rst

```diff
@@ -2,25 +2,7 @@
 Change Proposal Process
 =======================
 
-https://github.com/opentelekomcloud-docs/sandbox
-
-Editing Documentation
-=====================
-
-The documentation format we use is Restructured Text, often
-abbreviated "ReST" or ".rst" as a file extension. It is very similar
-to Markdown, but more powerful since it has some extra features. But
-basically, it is just plain unformatted text. Headlines can be done
-with a "#" or "##" in front of the title, bold text is produced by
-embracing the text with "**", italics are created by "__". There's a
-lot of external documentation on the details (https://sphinx-tutorial.readthedocs.io/step-1/).
-
-  * Toctree
-  * Links
-  * Makros
-  * Automatically generated documentation from source
-  * Editorial recommendations for documentation on language
-  * Building the documentation locally
+To apply a change in the documentation a number of steps have to be performed. They are listed on this page step by step.
 
 Initializing a repo
```

Comment:
I would recommend to rename the title here. We are not initializing a repo, but making fork for further work

Reply:
Also a hint that this is a one-time job